code/game/world.dm

```diff
@@ -328,7 +328,8 @@ GLOBAL_VAR(restart_counter)
 		hostedby = CONFIG_GET(string/hostedby)
 
 	s += "<b>[station_name()]</b>";
-	s += "(<a href='https://discord.gg/z9ttAvA'>Discord</a>|<a href='http://beestation13.com'>Website</a>)"
+	var/discordurl = CONFIG_GET(string/discordurl)
+	s += "(<a href='[discordurl]'>Discord</a>|<a href='http://beestation13.com'>Website</a>))"
```

Comment:
can the website be unhardcoded too?

Reply:
If you want to edit config, then yes

Reply:
We need to add new variable to config.
And then just take it out

Reply:
Or we can take lonk to the rules/forum